.github/workflows/ci.yml

```diff
@@ -39,3 +41,12 @@ jobs:
             http://localhost:8000/
             http://localhost:8000/listing
           lhConfigPath: ./.lighthouserc.js
+
+      - name: Release package
+        if: github.event_name == 'push'
+        run: npx @s-ui/ci release
+        env:
+          GITHUB_EMAIL: cloud-accounts@scmspain.com
```

Comment:
not sure this works out of the box but... could we keep this agnostic to our account email?

```suggestion
          GITHUB_EMAIL: ${{github.email}}
```

Reply:
Yes, we should, but... https://github.com/SUI-Components/sui-components/blob/master/.github/workflows/node.js.yml#L203